splink/files/settings_jsonschema.json

```diff
@@ -211,12 +237,6 @@
             "$id": "#/properties/comparison_columns/items/properties/gamma_index",
             "type": "integer",
             "description": "Gamma values in the comparison vector will be put in columns called gamma_0, gamma_1 etc.  This is the gamma index corresponding to this column"
-          },
```

Comment:
This was a previous attempt to account for custom columns that was never properly implemented so acn be deleted